model/src/main/proto/platform_parameter.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// It will correspond to a single Platform Parameter Value.
+message PlatformParameter {
+    // Value of a parameter can be of any one of the following type.
+    oneof value {
```

Comment:
Should be ``value_type`` per conventional use of oneofs elsewhere in the codebase.

Reply:
Done